packaging/templates/testnet-v4/sentry/sentry/bor/config.toml

```diff
@@ -41,12 +41,12 @@ syncmode = "full"
     # locals = []
     # journal = ""
     # rejournal = "1h0m0s"
-    # pricelimit = 30000000000
+    # pricelimit = 1
     # pricebump = 10
 
 [miner]
-    gaslimit = 20000000
-    # gasprice = "30000000000"
+    gaslimit = 30000000
+    # gasprice = "1000000000"
```

Comment:
Is this supposed to be commented?

Reply:
Yes, the gas price is defaulted to 1 Gwei only in the code (which is correct for mumbai). 

I have updated the commented values (even though it does not matter technically) because they denote the code defaults. Even though we're not using it, it just represents that if you don't pass, this value is getting used in the code.